includes/admin/class-wc-admin-attributes.php

```diff
@@ -202,7 +202,7 @@ private static function process_edit_attribute() {
 			);
 		}
 
-		echo '<div class="updated"><p>' . __( 'Attribute updated successfully', 'woocommerce' ) . '</p></div>';
+		echo '<div class="updated"><p>' . __( 'Attribute updated successfully', 'woocommerce' ) . '</p><p><a href="'. admin_url( 'edit.php?post_type=product&amp;page=product_attributes' ).'">&larr; ' . __( 'Back to Attributes', 'woocommerce' ) . '</a></p></div>';
```

Comment:
Should i replace `admin_url` with `esc_url` or can wrap `admin_url` into `esc_url`?